File: backend/app/models/coffee.py
from sqlalchemy import Column, Integer, String, Float, Boolean

from app.database.connection import Base


class Coffee(Base):
    __tablename__ = "coffees"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String, nullable=False)
    description = Column(String, nullable=True)
    price = Column(Float, nullable=False)
    category = Column(String, nullable=False)
    is_available = Column(Boolean, default=True)

File: frontend/src/types/coffee.ts
export interface Coffee {
  id: number;
  name: string;
  description: string;
  price: number;
  category: string;
  is_available: boolean;
}

export interface CoffeeCreate {
  name: string;
  description: string;
  price: number;
  category: string;
  is_available: boolean;
}